packages/cli/src/services/frontend.service.ts

```diff
@@ -277,7 +278,6 @@ export class FrontendService {
 
 		if (config.getEnv('nodes.communityPackages.enabled')) {
 			// eslint-disable-next-line @typescript-eslint/naming-convention
-			const { CommunityPackagesService } = await import('@/services/communityPackages.service');
```

Comment:
`CommunityPackagesService` is lazy loaded everywhere to avoid to reduce memory usage when the service isn't used.
By moving this out, that code is now always loaded.